subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ReportImplementation.kt

```diff
@@ -118,11 +114,11 @@ class ReportImplementation(
         exception: Throwable,
         screenshot: FutureValue<RemoteStorage.Result>?
     ) = methodExecutionTracing("registerIncident") {
-        val currentState = getCastedState<ReportState.Initialized>(cause = exception)
+        val currentState = getCastedState<Initialized>(cause = exception)
 
         if (currentState.incident == null) {
```

Comment:
Just want to notice a strange limitation. We can't register and compose multiple incidents in the report.